kratos/linear_solvers/fallback_linear_solver.h

```diff
@@ -0,0 +1,897 @@
+     * @param Settings A `Parameters` object containing the configuration settings for the linear solver to be constructed. This includes type, tolerance, maximum iterations, and other solver-specific parameters.
+     * @return Returns a `LinearSolverPointer` pointing to the newly constructed linear solver instance. If the factory implementation cannot find a matching solver type or if essential parameters are missing or invalid, it throws an exception.
+     * @exception std::invalid_argument Thrown if the settings specify an unsupported solver type or if required parameters are missing or invalid.
+     */
+    LinearSolverPointer ConstructLinearSolverFromSettings(const Parameters Settings)
+    {
+        // Implementation that delegates to a factory method.
+        // The factory method 'Create' is responsible for parsing 'Settings'
+        // and constructing the appropriate solver.
+        const auto linear_solver_factory = LinearSolverFactoryType();
+        Parameters linear_solver_settings(Settings);
+        const std::string linear_solver_type = linear_solver_settings["solver_type"].GetString();
+        // If the solver type is a faster direct solver, try to use it
+        if (linear_solver_type == "faster_direct_solver") {
+            std::vector<std::string> faster_direct_solvers;
+            if constexpr (TSparseSpaceType::IsDistributed()) {
+                faster_direct_solvers = std::vector<std::string>({ // May need to be updated and reordered. In fact I think it depends of the size of the equation system
+                    "mumps2",         // Amesos2 (if compiled with MUMPS-support)
+                    "mumps",          // Amesos (if compiled with MUMPS-support)
+                    "super_lu_dist2", // Amesos2 SuperLUDist (if compiled with MPI-support)
+                    "super_lu_dist",  // Amesos SuperLUDist (if compiled with MPI-support)
+                    "amesos2",        // Amesos2
+                    "amesos",         // Amesos
+                    "klu2",           // Amesos2 KLU
+                    "klu",            // Amesos KLU
+                    "basker"          // Amesos2 Basker
+                });
+            } else {
+                faster_direct_solvers = std::vector<std::string>({
+                    "pardiso_lu",              // LinearSolversApplication (if compiled with Intel-support)
+                    "pardiso_ldlt",            // LinearSolversApplication (if compiled with Intel-support)
+                    "sparse_lu",               // LinearSolversApplication
+                    "skyline_lu_factorization" // In Core, always available, but slow
```

Comment:
What do you mean @RiccardoRossi, skyline is the last option. Don't confuse an example given for understanding how it works with the actual code